webapp/src/blocks.tsx

```diff
@@ -908,9 +908,6 @@ export class Editor extends toolboxeditor.ToolboxEditor {
     public moveFocusToFlyout() {
         if (this.keyboardNavigation) {
             this.keyboardNavigation.focusFlyout();
-        } else {
-            // Move focus to the workspace. If Blockly change the flyout to be focussable this could focus it instead.
-            (this.editor.getInjectionDiv() as HTMLDivElement).focus();
```

Comment:
This was the previous MakeCode behaviour, but it's not a useful fallback behaviour where accessible blocks is enabled and harmful if it's not enabled.